src/appleseed.studio/mainwindow/rendering/lightpathswidget.cpp

```diff
@@ -61,21 +64,35 @@ using namespace std;
 namespace appleseed {
 namespace studio {
 
+namespace {
+    const Matrix4f get_gl_matrix(const Matrix4f& m)
```

Comment:
`get_gl_matrix` does not reflect what the function does. And by seeing the content of the function, you can remove it and call `transpose` directly.

Reply:
Fair enough, I think it semantically does match what the function does, at least in my mind, but you're right that it's quite simple and can probably be removed :)